### ISSUE TITLE
OperationalError: no such column: None

### ISSUE THREAD

**Issue by panomitrius** [degiz/kobo2calibre#2]
Running the code with the following line `poetry run python parse_kobo_highlights.py "/Volumes/KOBOeReader" "$HOME/Documents/Calibre Library"`  gives the following error message:

```
Traceback (most recent call last):
  File "/Users/panomitrius/kobo2calibre/parse_kobo_highlights.py", line 87, in <module>
    main(args)
  File "/Users/panomitrius/kobo2calibre/parse_kobo_highlights.py", line 35, in main
    likely_book_id, likely_book_path = db.get_likely_book_path_from_calibre(
  File "/Users/panomitrius/kobo2calibre/db.py", line 43, in get_likely_book_path_from_calibre
    book_path = cur.execute(f"SELECT path FROM books WHERE id = {book_id}").fetchone()[
sqlite3.OperationalError: no such column: None
```

**Comment by degiz** [comment]
Hey @panomitrius 

Were you able to resolve that?

**Comment by panomitrius** [comment]
No, I wasn't..

**Comment by dalmo1991** [comment]
I think this is caused when a book is in the Kobo but not in Calibre. Could it be the case?

**Comment by degiz** [comment]
This will be solved by a GUI calibre plugin that I'm working on now 👍

**Comment by degiz** [comment]
Hey @panomitrius 

I've just published [0.1 version](https://github.com/degiz/kobo2calibre/releases/tag/0.1), please give it a try!
I expect it to resolve the issue that you've had.
